packages/frontend/src/components/UserSettings/CreateGroupModal/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button, Group, Modal, Stack, TextInput, Title } from '@mantine/core';
+import { useForm } from '@mantine/form';
+import { IconUsersGroup } from '@tabler/icons-react';
+import React, { FC } from 'react';
+import { useGroupCreateMutation } from '../../../hooks/useOrganizationGroups';
+import { useApp } from '../../../providers/AppProvider';
+import MantineIcon from '../../common/MantineIcon';
+
+const CreateGroupModal: FC<{
+    opened: boolean;
+    onClose: () => void;
```

Comment:
I would pick from `ModalProps` 👍 you can just do: `FC<ModalProps>`

Reply:
👍